projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -2,16 +2,22 @@ import './editor.scss';
 
 import { useAnalytics } from '@automattic/jetpack-shared-extension-utils';
 import apiFetch from '@wordpress/api-fetch';
-import { useBlockProps } from '@wordpress/block-editor';
+import { useBlockProps, InnerBlocks } from '@wordpress/block-editor';
+import { pasteHandler } from '@wordpress/blocks';
 import { Placeholder, Button, Spinner } from '@wordpress/components';
-import { useSelect } from '@wordpress/data';
-import { useState, RawHTML, useEffect } from '@wordpress/element';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useState, useEffect, useCallback } from '@wordpress/element';
 import { sprintf, __ } from '@wordpress/i18n';
+import classNames from 'classnames';
+import { STATE as AI_BLOCK_STATE } from './attributes';
 import { name as aiParagraphBlockName } from './index';
 
 // Maximum number of characters we send from the content
 export const MAXIMUM_NUMBER_OF_CHARACTERS_SENT_FROM_CONTENT = 240;
 
+const is_untriggered = state => state === AI_BLOCK_STATE.DEFAULT;
+const is_triggered = state => state !== AI_BLOCK_STATE.DEFAULT;
```

Comment:
Probably not, but as it was one line and it made the code more readable I prefer it.  Also, there is a possibility that the logic might change. (For instance the untriggered state might actually need to be none of the defined states -- I saw a weird state happen when I was testing deprecations where this would have been a good fix)